src/v7.rs

```diff
@@ -41,15 +43,41 @@ impl Uuid {
     /// );
     /// ```
     ///
+    /// A v7 UUID can also be created with a counter to ensure batches of
+    /// UUIDs created together remain sortable:
+    ///
+    /// ```rust
+    /// # use uuid::{Uuid, Timestamp, Context};
+    /// let context = Context::new(42);
+    /// let uuid1 = Uuid::new_v7(Timestamp::from_unix(&context, 1497624119, 1234));
```

Comment:
These examples should use `Timestamp::from_unix_wide` with a 42bit counter to steer users wanting counters in the right direction.